docs/src/dataloader.md

```diff
@@ -168,3 +188,50 @@ public interface IUsersStore
 	Task<IEnumerable<User>> GetAllUsersAsync();
 }
 ```
+## Usage *without* Dependency Injection
+Because the DataLoaderContext is defined in the GraphQL Root object we can get it in our resolvers like this:
+```csharp
+public class ordertype : ObjectGraphType<order>
+{
+    public ordertype()
+    {
+        Name = "Ordertype";
+        UserStore store = new UserStore();
+
+        Field<IntGraphType>("id");
+        Field<StringGraphType>("userId");
+        Field<IntGraphType>("category");
+        Field<StringGraphType>("name");
+        Field<DateGraphType>("date");
+        Field<Usertype, User>.Name("user").ResolveAsync(ctx =>
+        {
+            store.SetResolveContext(ctx); //Make ctx availabele
```

Comment:
I don’t understand why you would need to set the context on the store. Your store shouldn’t need to know anything about GraphQL.